beacon_node/beacon_processor/src/lib.rs

```diff
@@ -1287,18 +1276,15 @@ impl<E: EthSpec> BeaconProcessor<E> {
                             // Attestation batches are formed internally within the
                             // `BeaconProcessor`, they are not sent from external services.
                             Work::GossipAttestationBatch { .. } => crit!(
-                                    self.log,
-                                    "Unsupported inbound event";
-                                    "type" => "GossipAttestationBatch"
+                                r#type = "GossipAttestationBatch",
```

Comment:
```suggestion
                                type = "GossipAttestationBatch",
```

Reply:
shouldn't we stick to using `r#type` here? ran into some issues while trying to use `type` here ,as it's a reserved keyword.
```suggestion
                                r#type = "GossipAttestationBatch",
```
Please lmk if there's any other way to handle this